buildSrc/subprojects/parser/src/main/java/org/apache/calcite/buildtools/parser/DialectTraverser.java

```diff
@@ -56,12 +57,13 @@ public DialectTraverser(File dialectDirectory, File rootDirectory,
   }
 
   /**
-   * Extracts functions and prints the results for the given dialect.
+   * Extracts functions and token assignments and prints the functions for the
+   * given dialect.
    */
   public void run() {
-    Map<String, String> functions = extractFunctions();
+    ExtractedData extractedData = extractData();
     // TODO(AndrewPochapsky): Remove this once generation logic added.
-    for (Map.Entry<String, String> entry : functions.entrySet()) {
+    for (Map.Entry<String, String> entry : extractedData.functions.entrySet()) {
```

Comment:
Should we also print out the tokens assignments?

Reply:
Sure, will add.